bridges/otellogr/logsink.go

```diff
@@ -201,9 +206,8 @@ var _ logr.LogSink = (*LogSink)(nil)
 // For example, commandline flags might be used to set the logging
 // verbosity and disable some info logs.
 func (l *LogSink) Enabled(level int) bool {
-	ctx := context.Background()
 	param := log.EnabledParameters{Severity: l.levelSeverity(level)}
-	return l.logger.Enabled(ctx, param)
+	return l.logger.Enabled(l.ctx, param)
```

Comment:
This change should also be in its own PR, as it's not needed for the panic fix, and it adds a new feature (`Enabled` can now look into context values).